WeScan/Review/ReviewViewController.swift

```diff
@@ -90,6 +104,8 @@ final class ReviewViewController: UIViewController {
     
     private func setupViews() {
         view.addSubview(imageView)
+        imageView.addSubview(imageCheck)
+        AudioServicesPlaySystemSound(kSystemSoundID_Vibrate)
```

Comment:
Why did you opt to use `AudioToolbox` over using haptics here? Could we instead consider using `UINotificationFeedbackGenerator`?